public/video-ui/src/util/isPublishable.js

```diff
@@ -0,0 +1,15 @@
+import { previewPrivacyState } from '../constants/privacyStates';
+
+export function isPublishable(video) {
```

Comment:
Bit of an odd name - I'd expect this to return a `bool` but it returns an `Object`?